README.md

```diff
@@ -12,7 +12,7 @@ In order to be able to get new builds of the DNX, and switch between them, you n
 
 ## Getting Started on Windows
 
-The easiest way to get started on Windows is to grab the latest preview of Visual Studio 2015, which can be found [here](http://go.microsoft.com/fwlink/?LinkId=521794).
+The easiest way to get started on Windows is to grab Visual Studio 2015 Community, which can be found [here](https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx).

```

Comment:
Oh I was thinking Enterprise and Community. But linking to VS Code ain't a bad idea either.

@DamianEdwards @danroth27 any thoughts on this one?